app/controllers/documents.js

```diff
@@ -0,0 +1,143 @@
+class Document{
+
+  constructor(){
+    this.jwt = require('jsonwebtoken');
+    this.config = require('./../config/config');
+    this.models = require('./../models/index');
+  }
+
+  createDoc(req, res){
+
+    const verifyToken = this.jwt.verify(req.headers.authorization, this.config.secrete);
+    this.models.documents.createDoc(req, verifyToken.id, verifyToken.role).then((document)=>{
+      res.status(201).json({
+        status: 'success',
+        message: 'Document created successfully',
+        data: document
+      });
+    }).catch((error)=>{
```

Comment:
Let's put `.catch` on a separate line. There are other occurrences in the rest of the code, please fix them for pattern consistency